pkg/authentication/firebase.go

```diff
@@ -48,3 +50,31 @@ func NewFirebaseWithTokenVerifier(firebaseAuth FirebaseTokenVerifier) Authentica
 		firebaseAuth: firebaseAuth,
 	}
 }
+
+// firebaseAuth uses the firebase application to refresh the keys used to verify the token signature.
+// The firebase.App has an internal mechanism to avoid repeating this operation for every request.
```

Comment:
I think that this can be updated to mention that it wraps a standard Firebase application object to implement internal interfaces.

I believe the second line can be removed.

Reply:
Makes a lot of sense!